### DISCUSSION TITLE
Install YOURLS in a subfolder

### DISCUSSION THREAD

**Discussion by lubrell** [YOURLS/YOURLS#3576] [General]
Hello all,

I would like to install YOURLS in a subfolder (www.example.com/yourls/), but still output the shortened URLs via the domain (www.example.com/shortenURL).

Is this possible? There is a website running under the domain and YOURLS should be used in private mode.

Thanks for your answers,

lubrell

**Comment by lawrence-eka** [comment]
Hi, I would like to know how to this, too.

**Comment by ozh** [comment]
https://yourls.org/docs/guide/server-configuration#give-yourls-its-own-directory
